### ISSUE TITLE
Detail for HME100k training

### ISSUE THREAD

**Issue by JackJi12316** [qingzhenduyu/TAMER#3]
Hi, thanks for your work!

In our experiment, the training HME100k dataset is not very effective, can you tell us more details about the training process? 

Thank you very much : D

**Comment by JackJi12316** [comment]
Oh, num_layer I set same as CROHME

**Comment by qingzhenduyu** [comment]
The train details can be found in `lightning_logs/version_1/config.yaml`.

**Comment by qingzhenduyu** [comment]
For model with fusion module, you can checkout to the 'fusion' branch. Our experiments show that TAMER with Fusion Module surpass all previous methdod.

**Comment by JackJi12316** [comment]
Thanks for your reply, did your experiment use 4 2080Ti, AdamW optimizer without schedular

**Comment by qingzhenduyu** [comment]
There is no big performance difference between AdamW or Adadelta, only need to change the lr. We used 4 2080Ti. to train our model.

**Comment by JackJi12316** [comment]
Thanks, we will try again, whether it will show better when use SGD with ReduceLROnPlateau shedular?
And wherther we can use pytorch lightning tool to initialize AdamW lr?
Thanks for you advise!

**Comment by qingzhenduyu** [comment]
1. Although training the model on HME100K using SGD with the ReduceLROnPlateau scheduler takes more time, the model typically achieves better performance under this setup.

2.

>  "And wherther we can use pytorch lightning tool to initialize AdamW lr?"

 Sorry, I don't quite understand what this question is asking. But if you want to change the lr, please modify `learning_rate `in `config/hme100k.yaml`

**Comment by JackJi12316** [comment]
Thank you very much for your answers, and thank you and your team for their contributions in this field!

Have a good day!
